src/signals/shared/filter/parse.js

```diff
@@ -64,6 +65,16 @@ export const parseOutputFormData = form => {
     name, refresh, id, ...options
   } = parsed;
 
+  // before/after params require a specific format that includes the time
+  // since `created_before` means before and including, we add a day
+  if (moment(options.created_before, 'YYYY-MM-DD').toISOString() !== null) {
+    options.created_before = moment(options.created_before).set({ hours: 23, minutes: 59, seconds: 59 }).format('YYYY-MM-DDTkk:mm:ss');
```

Comment:
Still not a fan of this.
Before should always mean the same. So if that mean we need to add or remove a day for displaying purposes, we should be doing that in my opinion.